lib/simulator-xcode-9.js

```diff
@@ -150,6 +150,29 @@ class SimulatorXcode9 extends SimulatorXcode8 {
           `'${prefs.SimulatorWindowLastScale}' is assigned instead.`);
       }
     }
+
+    if (!_.isUndefined(prefs.SimulatorWindowCenter)) {
+      const verificationPattern = /\{\-?\d+(\.\d+)?,\-?\d+(\.\d+)?\}/;
+      if (!_.isString(prefs.SimulatorWindowCenter) || !verificationPattern.test(prefs.SimulatorWindowCenter)) {
+        log.errorAndThrow(`SimulatorWindowCenter is expected to match "{floatXPosition,floatYPosition}" format (without spaces). ` +
```

Comment:
Why not accommodate spaces?

Reply:
because there are no spaces in the original plist, so I'm not sure if it works properly if I add some

Reply:
Ok. It might be good to allow them as input (even if the spaces need to be stripped out on the server side) since some language's serialization library might include spaces at some point. Perhaps that is better left for if/when such an event occurs. From the error message it would be obvious what the problem is.

Reply:
I expect the value to be passed as a simple string, so nothing is going to be changed by the language

Reply:
But if, in my test, I compute the value, put it in an object, and then serialize that, if the serialization adds a space it would not work.

But this is conjecture. I don't know of any particular JSON serializer that would do it, and generally they like to compress as much as possible.